src/foam/nanos/crunch/Capability.js

```diff
@@ -157,46 +158,6 @@ foam.CLASS({
       class: 'String',
       includeInDigest: true,
     },
-    {
-      name: 'expiry',
```

Comment:
this shouldn't be removed since the expiry and etc is used to set the expiry on ucjs

Reply:
in `ConfigureUCJExpiryOnGranted`

Reply:
'expiry' is provided by Renewable, which Capability now implements

Reply:
oops, yes, lgtm then